src/shared/cache/PmidCache.ts

```diff
@@ -2,34 +2,91 @@ import LazyMobXCache from "../lib/LazyMobXCache";
 import {AugmentedData} from "../lib/LazyMobXCache";
 import request from "superagent";
 
-async function fetch(pmids: number[])
+export type PmidData = {
+        [pmid:string]:PmidDatum;
+};
+
+export type PmidDatum = {
```

Comment:
Wow, thanks for doing this!

Can we move this one under model? I can update other references to use this model within the OncoKB related components later.